auth_keycloak/tests/test_wizard_sync.py

```diff
@@ -35,13 +34,16 @@ def test_get_token(self):
         request = responses.calls[0].request
         self.assertEqual(request.url, self.provider.auth_endpoint)
         expected = [
+            ('grant_type', 'password'),
```

Comment:
this important change is for? :smile:

Reply:
It's an artifact of me trying to make the test work. `assertListEqual` also takes into account the order of the items. I first just tried to change the order to the result, but that didn't work :P So I used sorted and that worked.

Reply:
I see, than make it explicit and fail proof by using `expected = sorted([...` so that no matter who adds what and in which order ;)

Reply:
as you did already... :smile: